### ISSUE TITLE
Accessibility issues with aria-hidden set to true

### ISSUE THREAD

**Issue by schrenkk** [lifarl/react-scroll-snap-slider#36]
There is currently an accessibility violation caused by the scroll snap slider. See the image:

Have a look at https://github.com/lifarl/react-scroll-snap-slider/blob/master/src/components/Carousel/index.tsx#L80 where the aria-hidden attribute is set to true. The violation indicated by Storybook is that the content on this slide is still focusable, which is not allowed.

To solve this issue, we need to set the slide to hidden. This could be done like this:
```
slideRefs.current[index]?.setAttribute('style', 'visibility: hidden');
````

However, maybe we should try to avoid using the style attribute and instead use a class for this?

Have a look at the best practices from w3.org: The solution comes from https://www.w3.org/WAI/tutorials/carousels/working-example/

Another question is whether this repository is still maintained and if PRs are welcome. I didn't find any contribution guide, and the build job is currently failing on my local machine.
